TestAssets/TestProjects/DesktopUsingPackageWithSatellites/DesktopUsingPackageWithSatellites.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
I've added the coverage in 7ba7a66 to be safe. I can't see from code inspection how `TargetFramework` vs `TargetFrameworks` would make any difference so I suspect things were conflated.